hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -0,0 +1,142 @@
+
+:::image type="content" source="images/hello-world/configure-project.png" alt-text="Specify project details":::
+
+Create a new C# solution using the **Uno Platform App** type from Visual Studio's **Start Page**. You may need to choose a different name for your solution.
+
+Now you'll choose a base template to take your Hello World application multi-platform. The Uno Platform App template comes with two preset options that allow you to quickly get started with either a Blank solution, or the Default configuration which includes references to the Uno.Material and Uno.Toolkit libraries, Uno.Extensions is used for dependency injection, configuration, navigation and logging, and it uses MVUX in place of MVVM, making it a great starting point for rapidly building real world applications. 
+
+To keep things simple, select the **Blank** preset.
+
+:::image type="content" source="images/hello-world/vsix-new-project-options.png" alt-text="Uno solution template for project startup type":::
+
+Click the **Create** button. Wait for the projects to be created and their dependencies to be restored.
+
+A banner at the top of the editor may ask to reload projects, click **Reload projects**:
+:::image type="content" source="images/hello-world/vs2022-project-reload.png" alt-text="Visual Studio banner offering to reload your projects to complete changes":::
+
+## Building your app
+
+Now that you've generated the functional starting point of your multi-platform WinUI application, you can copy markup into it from the Hello World WinUI 3 project outlined in the [previous](/hub/apps/how-tos/hello-world-winui3.md) tutorial.
+
+You should see the following default file structure in your Solution Explorer:
+
+:::image type="content" source="images/hello-world/uno-file-structure.png" alt-text="Default file structure":::
+
+Make sure Visual Studio has your WinUI 3 project open, then copy the child XAML elements from `MainWindow.xaml` in the WinUI 3 project to your `MainPage.xaml` file in the Uno Platform project. The `MainPage` view XAML should look like this:
+
+```xml
+<Page x:Class="HelloWorld.MainPage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:HelloWorld"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  mc:Ignorable="d"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+      <!-- Below is the code you copied from MainWindow: -->
+      <StackPanel Orientation="Horizontal"
+                  HorizontalAlignment="Center"
+                  VerticalAlignment="Center">
+          <TextBlock x:Name="myText" 
+                     Text="Hello world!"
+                     Foreground="Red"/>
+      </StackPanel>
+</Page>
+```
```

Comment:
Run the WinUI target first and comment that it looks the same as the WinUI app from previous tutorial